### ISSUE TITLE
I have a question: if we already have the ground-truth image generated from "non-diffusive generators", why still use diffusion part? Which purpose of this action? Thank u!

### ISSUE THREAD

**Issue by Wangzs0228** [icon-lab/SynDiff#3]

**Comment by onat-dalmaz** [comment]
Hello,
The images that are generated via the diffusive module have higher sample fidelity and diversity. Thus, instead of the images generated via non-diffusive generators, we sample from the diffusive generators.

**Comment by Wangzs0228** [comment]
> Hello, The images that are generated via the diffusive module have higher sample fidelity and diversity. Thus, instead of the images generated via non-diffusive generators, we sample from the diffusive generators.

Thank u for your reply. So, to be summary, the diffusive generators could be treated as an image enhancer? It still a supervised module? Is that right?

**Comment by onat-dalmaz** [comment]
Yes, that's correct. The diffusive generators can be thought of as an image enhancer, because they improve the quality and diversity of the images generated by the non-diffusive generators. The diffusive generators can be considered a supervised module because they are trained using image estimates from the non-diffusive module.
